Eventnet/Eventnet.Api/Services/SaveServices/EventSaveService.cs

```diff
@@ -17,21 +17,25 @@ public EventSaveService(IPublishEventService publishEventService, Handler handle
 
     public async Task SaveAsync(Event eventForSave, IFormFile[] photos)
     {
+        var saveEventResult = new SaveEventResult(EventSaveStatus.InProgress, string.Empty);
+        handler.Update(eventForSave.Id, saveEventResult);
         try
         {
-            var streams = await GetRabbitMqPhotosAsync(photos);
-            var message = JsonSerializer.Serialize(new RabbitMqMessage(eventForSave, streams));
-            var saveEventResult = new SaveEventResult(EventSaveStatus.InProgress, string.Empty);
-            handler.Update(eventForSave.Id, saveEventResult);
+            var rabbitMqPhotos = await GetRabbitMqPhotosAsync(photos);
+            var message = JsonSerializer.Serialize(new RabbitMqMessage(eventForSave, rabbitMqPhotos));
             await publishEventService.PublishAsync(message);
         }
         catch (Exception e)
         {
+            saveEventResult = new SaveEventResult(EventSaveStatus.NotSavedDueToServerError, string.Empty);
+            handler.Update(eventForSave.Id, saveEventResult);
             Console.WriteLine(e);
             throw;
         }
     }
 
+    public bool ContainsGuid(Guid id) => handler.ContainsGuid(id);
+
```

Comment:
Плохой нейминг. В одних методах уточняешь, что сущность относится к Event (GetSaveEventResult), в других - считаешь так по-умолчанию (SaveAsync, ContainsGuid).

```suggestion
    public bool IsSaving(Guid eventId) => handler.ContainsGuid(id);

```